src/items/implementations.md

```diff
@@ -50,10 +50,44 @@ bracketed set of associable items.
 The nominal type is called the _implementing type_ and the associable items are
 the _associated items_ to the implementing type.
 
-Inherent implementations associate the contained items to the implementing type.
-The associated item has a path of a path to the implementing type followed by
-the associate item's path component. Inherent implementations cannot contain
-associated type aliases.
+Inherent implementations associate the contained items to the
+implementing type.  Inherent implementations can contain [associated
+functions] (including methods), [associated types], and [associated
+constants]. They cannot contain associated type aliases.
+
+[associated functions]: associated-items.html#associated-functions-and-methods
+[associated types]: associated-items.html#associated-types
+[associated constants]: associated-items.html#associated-constants
+
+The path to an associated item is: any path to the implementing type,
+followed by the path to the associate item within the inherent
+implementation.
+
+Note that the path to the module containing the inherent
```

Comment:
I'm not quite sure we want to have this note here. It's a restatement of the (visibility chapter)[https://doc.rust-lang.org/nightly/reference/visibility-and-privacy.html].

Reply:
That's fair.  I haven't read the whole reference yet, but it occurred to me as a subtle consequence that was implied by what was previously there.  Maybe a link to the visibility chapter would be more appropriate than restating?

Reply:
I removed the note paragraph and instead called it out in the example by showing side by side that impl blocks in different modules still use a path to the type.